### ISSUE TITLE
benchmark against other modules

### ISSUE THREAD

**Issue by y** [andygrundman/Image-Scale#3]
Besides GD, you also listed Image::Magick and Imager in the SEE ALSO section of Image::Scale pod. But you didn't include them in the benchmarks. There are several other modules that could be added as well, such as Image::Imlib2, Image::Resize, Image::Resize::OpenCV and PDL.

**Comment by andygrundman** [comment]
Yes, you're right, it should be benchmarked against similar tools. I wrote Image::Scale back in 2010 and at the time GD/IM were the two big ones I was aware of. The current TODO file contains a list of other libraries to research, I added the ones you mentioned, except Image::Resize which is just a GD wrapper.

Some other (non-Perl) libraries my coworkers mentioned:
http://gmic.eu/ (based on http://cimg.eu/)
https://www.lrde.epita.fr/wiki/Olena
http://www.vips.ecs.soton.ac.uk/index.php?title=VIPS
http://core.tcl.tk/akupries/crimp/wiki?name=References

Keep in mind that the original goal of Image::Scale was to achieve reasonable performance on slow embedded CPUs, and AFAIK no other open-source image library will come close to it on these systems. The fact that I::S is also faster on a normal system is just icing on the cake. :)
